src/main/java/com/robosh/ejournal/data/dto/UserDto.java

```diff
@@ -0,0 +1,30 @@
+package com.robosh.ejournal.data.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto {
+
+    protected Long id;
+
+    @NotBlank
+    protected String firstName;
+
+    @NotBlank
+    protected String secondName;
+
+    protected String lastName;
+
+    @Pattern(regexp = "[A-Za-z0-9+_.-]+@[a-z.-]+\\.[a-z]{2,8}")
+    protected String email;
+
+    @Pattern(regexp = "^\\+?3?8?(0\\d{9})$")
+    protected String phone;
```

Comment:
We can also create a custom annotation to check phone number

Reply:
i`m not sure it`s part of my task